src/tests/unit/DocumentSelectionHandler.test.ts

```diff
@@ -44,6 +44,7 @@ describe("DocumentSelectionHandler", () => {
 		DocumentSelectionRequestProcessor.getInstance = jest.fn().mockReturnValue(mockDocumentSelectionRequestProcessor);
 
 		await lambdaHandler(VALID_REQUEST, "");
+		// eslint-disable-next-line @typescript-eslint/unbound-method
```

Comment:
is this needed?

Reply:
not part of this PR commit, issue was with rebase with main.